handlers/infobip/infobip.go

```diff
@@ -218,13 +208,18 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 	}
 
 	// build our request
+	var authorization = fmt.Sprintf("App %s", msg.Channel().StringConfigForKey(courier.ConfigAPIKey, ""))
+	if sendURL == "" {
```

Comment:
Looks like this condition is always `true`. Can this part of the code be rewritten without using the if-statement?

Reply:
Yes, that was on purpose, see line 19 for the reason and see where it was used [here ](https://github.com/greatnonprofits-nfp/courier/blob/main/handlers/infobip/infobip_test.go#L218)

Reply:
@musamusa - I think Ihor refers to removing the conditional, as it's gonna be always true. So it would be only:
```
urlFromConfig := msg.Channel().StringConfigForKey(courier.ConfigSendURL, "")
sendURL = fmt.Sprintf("%s/sms/2/text/advanced", urlFromConfig)
```